# Single-cell-tutorials
> single-cell RNA seq tutorials for begginers

I am recently starting to work with single-cell analyses, something that it has been taking me a while since I have been wandering on the internet attempting to learn the basics. I decided to create this post to share with you some tutorials that have been really useful for me as a begginer. Please, feel free to contribute or give any feedback. 


<!-- TABLE OF CONTENTS -->
## Table of Contents

* [About](#About)
* [Python tutorials](#Python-tutorials)
* [R tutorial](#R-tutorial)
* [Other tutorials](#Other-tutorials)
* [Contact](#Contact)
* [Acknowledgements](#acknowledgements)


## About

## Python tutorials

* [Scanpy](https://scanpy.readthedocs.io/en/stable/#) a toolkit for preprocessing, visualizing, clustering, trajectory inference and differential expression testing of scRNA-seq data.
* [Single-cell-tutorial](https://github.com/theislab/single-cell-tutorial)(https://doi.org/10.15252/msb.20188746). Current best-practices in single-cell RNA-seq: a tutorial.

* [kallisto | bustools](https://www.kallistobus.tools/) a workflow for pre-processing single-cell RNA-seq data.

* [pySCENIC](https://pyscenic.readthedocs.io/en/latest/tutorial.html) a number of tutorials available to demonstrate how to use pySCENIC.


## R tutorials
[Seurat](https://satijalab.org/seurat/) an R package designed for QC, analysis, and exploration of single-cell RNA-seq data.


## Other tutorials
[Scanpy in R](https://theislab.github.io/scanpy-in-R/#content) a combination of scanpy and R strategies.


## Contact
If you want to contribute to this list or you have any issue regarding it, do not hesitate to contacte me.                            
Noel: ncabanas@iibiomedicas.unam.mx


## Acknowledgements
If the tutorials are useful please acknowledge the work of their creatrors citing their papers.
You can find how to cite them in the links.


